_includes/cards/event.html

```diff
@@ -0,0 +1,12 @@
+<div id="{{event.slug}}" class="card event" style="--banner: url('/assets/past_events/{{ event.slug }}.png')">
```

Comment:
magic

Reply:
Welcome to CSS vars =D https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_variables